docs/MANUAL_SCHEDULER_CONFIGURATION.md

```diff
@@ -127,6 +127,44 @@ following manual steps are required. Note that user must wait for all scheduler
     * Follow instructions for vanilla Kubernetes, but use the following path to the scheduler configuration file
     `/var/lib/rancher/rke2/agent/pod-manifests/`
 
+* K3S
+    * For manual patching folow next instructions  
+      * Create next configuration file in directory `/var/lib/rancher/k3s/agent/pod-manifests/scheduler`:
+        
+        ```
+        apiVersion: kubescheduler.config.k8s.io/v1beta1
```

Comment:
Please add that we should use `kubescheduler.config.k8s.io/v1beta3` for k8s 1.23 and above